sys/netpfil/ipfilter/netinet/ip_state.c

```diff
@@ -2207,20 +2206,6 @@ ipf_state_tcpinwindow(fr_info_t *fin, tcpdata_t  *fdata, tcpdata_t *tdata,
 		   (ackskew >= -1) && (ackskew <= 1)) {
 		inseq = 1;
 	} else if (!(flags & IS_TCPFSM)) {
-		int i;
-
-		i = (fin->fin_rev << 1) + fin->fin_out;
-
-#if 0
-		if (is_pkts[i]0 == 0) {
-			/*
-			 * Picking up a connection in the middle, the "next"
-			 * packet seen from a direction that is new should be
-			 * accepted, even if it appears out of sequence.
-			 */
-			inseq = 1;
-		} else
-#endif
```

Comment:
Good catch. This was removed by FreeBSD in IPF 4.2.28 but crept back in with IPF 5.1.2 as a mismerge.